### ISSUE TITLE
make error, No such file or directory

### ISSUE THREAD

**Issue by luojiah3** [LINKIWI/modern-paste#16]
I followed the installation instructions, this error occurred during _make_
```
make default
make[1]: Entering directory `/root/modern-paste'
mkdir -p app/static/build/js
mkdir -p app/static/build/css
python build/build_database.py --create
Creating database and all tables
python build/build_js.py --config-environment
Configuration in config.py specified a prod environment.
Compiling Javascript with default optimizations.
Javascript build started.
Compiling app/static/js/user/LoginController.js --> app/static/build/js/user/LoginController.js
Traceback (most recent call last):
  File "build/build_js.py", line 107, in <module>
    '--js_output_file', js_output_file,
  File "/usr/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
make[1]: *** [default] Error 1
make[1]: Leaving directory `/root/modern-paste'
make: *** [all] Error 2
```
**NOTE:** Before I encountered this error, I had to replace _modern-paste.git_ with _modern-paste_ in line 53 of tests/test_views/test_misc.py to pass _test_version_.

**Comment by luojiah3** [comment]
Same as #13

**Comment by LINKIWI** [comment]
The fixes of #12 and #13 will be merged into master soon.
